lib/API/JsonApi/v1/Filters/Interpreters/FilterInterpreter.php

```diff
@@ -348,4 +354,35 @@ protected function getOperator($operator)
             'containers '
         );
     }
+
+    /**
+     * @param string $field
+     * @return bool
+     */
+    protected function isCustomField($field)
+    {
+        if(!is_string($field)) {
+            throw new InvalidArgumentException('isCustomField requires $field to be a string');
+        }
+
+        return StringValidator::endsWith($field, '_c');
```

Comment:
Correct me if I'm wrong, but I think not all custom fields end with '_c' ? They probably should, but this isn't enforced. I think this function is going to miss a lot of custom fields.